instance/editions.go

```diff
@@ -38,8 +38,21 @@ func (s *Store) confirmEdition(ctx context.Context, datasetID, edition, instance
 			editionDoc = models.CreateEdition(s.Host, datasetID, edition)
 			log.Debug("created new edition", logData)
 		} else {
-			log.Debug("edition found, updating", logData)
+
 			action = UpdateEditionAction
+
+			if editionDoc.Current == nil && editionDoc.Next == nil {
```

Comment:
should this check more specifically that `.Links` exists? if the whole editions doc is empty thats definitely an issue, but the log message specifically talks about instance links